test/unit/specs/connectors/api.spec.js

```diff
@@ -98,6 +98,20 @@ describe(`API`, () => {
       })
     })
 
+    it(`queries a transaction by height`, async () => {
+      axios.mockReturnValue({})
+      await client.getTxsByHeight(`2`)
+      expect(axios.mock.calls).toEqual([
+        [
+          {
+            data: undefined,
```

Comment:
is this really expected?

Reply:
we're just testing that the right endpoint is called, not its returned value